mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule PlugLoggerJson.Mixfile do
       source_url: "https://github.com/bleacherreport/plug_logger_json",
       start_permanent: Mix.env == :prod,
       test_coverage: [tool: ExCoveralls],
-      version: "0.6.0",
+      version: "0.7.0",
```

Comment:
Can you remove this from the PR? We'll update this when we are ready to publish a new version.